src/security/ServerOptions.cc

```diff
@@ -17,13 +17,10 @@
 #include "SquidConfig.h"
 #if USE_OPENSSL
 #include "ssl/support.h"
```

Comment:
Thank you. 

[rant]You cannot really know for sure whether `openssl/err.h` needs our shim because both may change outside your control. It would be nice to be able to rely on the current state of code (and on folks carefully checking all side effects of their future changes), but that virtually never works long-term. Thus, when we can, we should use future-proof rules that are easy to apply/review and are programmatically verifiable. We are lucky that the rule we (mostly you) came up with during this PR is such a rule.[/rant]